frontend/src/components/image/ImageSwipe.vue

```diff
@@ -0,0 +1,292 @@
+            <div class="pswp__scroll-wrap">
+                <div class="pswp__container">
+                    <div class="pswp__item"></div>
+                    <div class="pswp__item"></div>
+                    <div class="pswp__item"></div>
+                </div>
+                <div class="pswp__ui pswp__ui--hidden">
+
+                    <div class="pswp__top-bar">
+                        <div class="pswp__counter"></div>
+                        <button class="pswp__button pswp__button--close" title="Close (Esc)"></button>
+
+                        <span class="rotation-wrapper">
+              <i class="material-icons" v-if=options.rotationOn @click="rotate(-90)">rotate_left</i>
+              <i class="material-icons" v-if=options.rotationOn @click="rotate(90)">rotate_right</i>
+            </span>
+
+                        <button class="pswp__button pswp__button--share" title="Share"></button>
+                        <button class="pswp__button pswp__button--fs" title="Toggle fullscreen"></button>
+                        <button class="pswp__button pswp__button--zoom" title="Zoom in/out"></button>
+                        <div class="pswp__preloader">
+                            <div class="pswp__preloader__icn">
+                                <div class="pswp__preloader__cut">
+                                    <div class="pswp__preloader__donut"></div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="pswp__share-modal pswp__share-modal--hidden pswp__single-tap">
+                        <div class="pswp__share-tooltip"></div>
+                    </div>
+                    <button class="pswp__button pswp__button--arrow--left" title="Previous (arrow left)" @click='resetAngle'>
+                    </button>
+                    <button class="pswp__button pswp__button--arrow--right" title="Next (arrow right)" @click='resetAngle'>
+                    </button>
+                    <div class="pswp__caption">
+                        <div class="pswp__caption__center"></div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    import PhotoSwipe from 'photoswipe/dist/photoswipe'
+    import PhotoSwipeUI_Default from 'photoswipe/dist/photoswipe-ui-default'
+    import 'photoswipe/dist/photoswipe.css'
+    import 'photoswipe/dist/default-skin/default-skin.css'
+
+    export default {
+        name: "ImageSwipe",
+        props: {
+            items: {
+                type: Array
+            },
+            options: {
+                default: () => ({}),
+                type: Object
+            }
+        },
+        data() {
+            return {
+                angle: 0
+            };
+        },
+        mounted() {
+            let that = this;
+            let initPhotoSwipeFromDOM = function (gallerySelector) {
```

Comment:
It is the worst approach ever, rewrite it in order to use virtual DOM of vue, you can't use operation with DOM of browser directly